django_tasks/__init__.py

```diff
@@ -38,7 +42,7 @@ def configure_settings(self, settings: Optional[dict]) -> dict:
             # HACK: Force a default task backend.
             # Can be replaced with `django.conf.global_settings` once vendored.
             return {
-                "default": {
+                DEFAULT_TASK_BACKEND_ALIAS: {
```

Comment:
Suggestion: As discussed, we should document that it should be called "default" (as with `DATABASES`).

Reply:
But even if discarding the rest of the PR, we could still reference that variable instead of writing `default` again, right?